cmake/submodule_configs.cmake

```diff
@@ -0,0 +1,52 @@
+macro(singularity_cmake_config pkg)
```

Comment:
This is kinda gross, but I see the need.

Also copyright notice.

Reply:
These have CR, on the latest push